static/app/views/issueDetails/groupRelatedIssues/index.tsx

```diff
@@ -70,107 +67,90 @@ function GroupRelatedIssues({params}: Props) {
+                    orgSlug={orgSlug}
+                    queryParams={{query: `issue.id:[${sameRootCauseIssues}]`}}
+                    query=""
+                    source="related-issues-tab"
+                    canSelectGroups={false}
+                    withChart={false}
+                  />
+                </div>
+              ) : (
+                <small>{t('No same-root-cause related issues were found.')}</small>
+              )}
+            </HeaderWrapper>
+          </div>
+          <div>
+            <HeaderWrapper>
+              <Title>{t('Issues in the same trace')}</Title>
+              {traceConnectedIssues.length > 0 ? (
+                <div>
+                  <TextButtonWrapper>
+                    <small>
+                      {t('These issues were all found within ')}
+                      <Link
+                        to={`/organizations/${orgSlug}/performance/trace/${traceMeta.trace_id}/?node=error-${traceMeta.event_id}`}
                       >
-                        {t('Open in Issues')}
-                      </LinkButton>
-                    </TextButtonWrapper>
-                    <GroupList
-                      endpointPath={`/organizations/${orgSlug}/issues/`}
-                      orgSlug={orgSlug}
-                      queryParams={{query: `issue.id:[${traceConnectedIssues}]`}}
-                      query=""
-                      source="related-issues-tab"
-                      canSelectGroups={false}
-                      withChart={false}
-                    />
-                  </div>
-                ) : (
-                  <small>{t('No trace-connected related issues were found.')}</small>
-                )}
-              </HeaderWrapper>
-            </div>
+                        {t('this trace')}
+                      </Link>
+                      .
+                    </small>
+                    <LinkButton
+                      to={`/organizations/${orgSlug}/issues/?query=trace:${traceMeta.trace_id}`}
+                      size="xs"
+                    >
+                      {t('Open in Issues')}
+                    </LinkButton>
+                  </TextButtonWrapper>
+                  <GroupList
+                    endpointPath={`/organizations/${orgSlug}/issues/`}
+                    orgSlug={orgSlug}
+                    queryParams={{query: `issue.id:[${traceConnectedIssues}]`}}
+                    query=""
+                    source="related-issues-tab"
+                    canSelectGroups={false}
+                    withChart={false}
+                  />
+                </div>
+              ) : (
+                <small>{t('No trace-connected related issues were found.')}</small>
+              )}
+            </HeaderWrapper>
           </div>
-        )}
-      </Layout.Main>
-    </Layout.Body>
-  );
-}
-
-function GroupRelatedIssuesWrapper(props: Props) {
-  return (
-    <Feature features={['related-issues']}>
```

Comment:
This has been moved to the Similar Issues main file:
<img width="465" alt="image" src="https://github.com/getsentry/sentry/assets/44410/5ab98ae2-8f93-43d7-972a-24a56836c31a">